### ISSUE TITLE
how to generate a document in symfony 4.1 with maker command

### ISSUE THREAD

**Issue by christiankasse** [symfony/maker-bundle#320]
Hello everyone, I am unable to generate documents with symfony 4.1 using mongodb because there are no commands for that, someone can help me please. thx

**Comment by bestog** [comment]
Wanted to ask the same thing. @weaverryan Is there a plan for that? 👍

**Comment by weaverryan** [comment]
No plan, but it doesn’t mean it can’t haopen. The make:entity command is super complex and took a long time to develop. Making one for th ODM, which would be great, is just much lower on the priority list because of the comparative number of users. Someone can contribute it of course.

Cheers!

**Comment by christiankasse** [comment]
And for the time what's can we do ? For moment a generate manually the getters and setters

**Comment by weaverryan** [comment]
Yep - for now, you’ll need to do things by hand like always - if you want getters and setters, using your IDE is a nice way (it sounds like you already are).

**Comment by SamuelTS** [comment]
> Hello everyone, I am unable to generate documents with symfony 4.1 using mongodb because there are no commands for that, someone can help me please. thx

I made a [composer package](https://packagist.org/packages/samuel-tallet/symfony-mongodb-document-maker) to handle this task. Project is still very young but basic features are there. 😉
